modules/apps/headless-apio/structured-content/structured-content-apio-impl/src/main/java/com/liferay/structured/content/apio/internal/architect/resource/StructuredContentNestedCollectionResource.java

```diff
@@ -349,6 +343,44 @@ private String _getLocalizedString(
 		return new PageItems<>(journalArticleWrappers, count);
 	}
 
+	private String _getRenderedContent(
+		JournalArticleWrapper journalArticleWrapper, DDMTemplate ddmTemplate,
+		Locale locale) {
+
+		return Try.fromFallible(
```

Comment:
use journalContent instead of service because we need to specify the DDM Template as a parameter.
use Try.fromFallible to prevent internal error